### PR TITLE
Prepares 0.2.1

### PR DESCRIPTION
**Pull request by iambmelt**
Closes https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/420

### CODE REVIEW

File: msal/build.gradle [hunk 1]
```diff
@@ -125,9 +125,9 @@ dependencies {
         transitive = false
     }
 
-    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '0.0.5', changing: true)
+    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '0.0.6', changing: true)
```
Comment (kreedula) [inline msal/build.gradle]: the common version here   should be `0.0.7` for the change

File: msal/build.gradle [hunk 2]
```diff
@@ -125,9 +125,9 @@ dependencies {
...
 
-    distApi("com.microsoft.identity:common:0.0.5") {
+    distApi("com.microsoft.identity:common:0.0.6") {
```
Comment (kreedula) [inline msal/build.gradle]: same here : `0.0.7`

File: msal/build.gradle (hunk 1 above)
Comment (iambmelt) [reply msal/build.gradle]: Thanks, resolved in d0bb6d1045c8c80f3aacb4b054ed45cf0c9cce0c

File: msal/build.gradle (hunk 2 above)
Comment (iambmelt) [reply msal/build.gradle]: Thanks, resolved in d0bb6d1045c8c80f3aacb4b054ed45cf0c9cce0c

**Comment by iambmelt** [comment]
Thanks #402 -- I'll merge once CI passes. Feel free to initiate deployment of `common@0.0.7`

File: changelog [hunk 3]
```diff
@@ -1,5 +1,11 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 
+Version 0.2.1
+----------------------------
+- Picks up common@0.0.6
```
Comment (Glennmen) [inline changelog]: Shouldn't this be `common@0.0.7`?

Comment (iambmelt) [reply changelog]: Yes, it should!

Comment (iambmelt) [reply changelog]: Fixed in 94c9463538127946179308626e8452dd20abe50d

**Pull request merged**